minio.json

```
@@ -29,23 +34,20 @@
                     }
                 },
                 "cmd_arguments": [
-                    [
-                        "server",
-                        "/data"
-                    ]
-                ],
+                    [ "server" , "/data" ]
+                 ],
                 "environment": {
-                    "MINIO_ACCESS_KEY": {
-                        "description": "MinIO Access Key (admin username), 1 to 128 alphanumeric characters",
-                        "label": "MinIO Access Key"
+                    "MINIO_ROOT_USER": {
+                        "description": "MinIO Administrator Username, 1 to 128 alphanumeric characters",
+                        "label": "MinIO Admin User"
                     },
-                    "MINIO_SECRET_KEY": {
-                        "description": "MinIO Secret Key (admin password), 1 to 128 alphanumeric characters",
-                        "label": "MinIO Secret Key"
+                    "MINIO_ROOT_PASSWORD": {
+                        "description": "MinIO Administrator Password, 1 to 128 alphanumeric characters",
```

```suggestion
                        "description": "MinIO Administrator Password, 8 to 128 alphanumeric characters",
```

@MikeMcPherson This was the reason it didn't work for me the first time. Sorry but I mixed up my messaging on my first test run and copied in the settings used for the failed Pi4 install (other reasons) but for the x86_64 install I had actually used a quick and dirty 4 letter pass and got the following in logs there after:
'''
Apr 06 14:26:36 rleap15-3 minio/minio:latest/MinIO[13506]: ERROR Unable to validate credentials inherited from the shell environment: Invalid credentials
Apr 06 14:26:36 rleap15-3 minio/minio:latest/MinIO[13506]:       > Please provide correct credentials
Apr 06 14:26:36 rleap15-3 minio/minio:latest/MinIO[13506]:       HINT:
Apr 06 14:26:36 rleap15-3 minio/minio:latest/MinIO[13506]:         Access key length should be at least 3, and secret key length at least 8 characters
'''